records_mover/db/redshift/loader.py

```diff
@@ -79,7 +79,19 @@ def load(self,
                                empty_as_null=True,
                                **redshift_options)  # type: ignore
             logger.info(f"Starting Redshift COPY from {directory}...")
-            self.db.execute(copy)
+            redshift_pid: int = self.db.execute("SELECT pg_backend_pid();").scalar()
+            try:
+                self.db.execute(copy)
+            except sqlalchemy.exc.InternalError:
+                # Upon a load erorr, we receive:
+                #
+                #  sqlalchemy.exc.InternalError:
+                #  (psycopg2.errors.InternalError_) Load into table 'tablename'
+                #    failed.  Check 'stl_load_errors' system table for details.
+                logger.warning("Caught load error - "
```

Comment:
❤️

Reply:
Yeah, I've been wanting this forever - happy to have figured out the pattern to make it happen!